.storybook/webpack.config.js

```diff
@@ -6,31 +6,58 @@
 const path = require('path')
 const webpack = require('webpack')
 const fs = require('fs')
+const config = require('../build/commands/lib/config')
+const genTsConfig = require('../build/commands/lib/genTsConfig')
 const { fallback, provideNodeGlobals } = require('../components/webpack/polyfill')
+const generatePathMap = require('../components/webpack/path-map')
 
-const buildConfigs = ['Component', 'Static', 'Debug', 'Release']
-const extraArchitectures = ['arm64', 'x86']
+// Choose which brave-core build directory to look for pre-compiled
+// resource dependencies:
+// 1. Default for local builds for the actual platform / architecture
+// 2. platform / architecture overriden by environment variables
+// 3. most recently built - this caters to the common scenario when a
+// non-standard target has been built but no arguments are provided to storybook
+config.update({
+  target_arch: process.env.TARGET_ARCH,
+  target_os: process.env.TARGET_OS,
+  target_environment: process.env.TARGET_ENVIRONMENT,
+  target: process.env.TARGET
+})
 
-function getBuildOuptutPathList(buildOutputRelativePath) {
+let outputPath = config.outputDir
+
+function getBuildOuptutPathList() {
   return buildConfigs.flatMap((config) => [
```

Comment:
Is `buildConfigs` defined here? :thinking: Perhaps I'm missing something.

Reply:
Looks like I deleted it by accident and didn't notice because the config's default output dir exists for me.
That's what we get for late night vacation  emergency fix!